spec/controllers/api/v1/orders_packages_controller_spec.rb

```diff
@@ -186,7 +186,7 @@
           it 'fails to dispatch the packages' do
             expect(current_state).to eq('designated')
 
-            put :exec_action, id: orders_package.id, action_name: 'dispatch'
+            put :exec_action, id: orders_package.id, action_name: 'dispatch', quantity: 10, location_id: package.locations.first
```

Comment:
Are we passing a location object for location_id? 🤔

Reply:
should be the id, I guess rails is being smart